test/api/schemas/response/keyword/success.json

```
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string"
+            },
+            "id": {
+              "type": "string"
+            },
+            "attributes": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string"
```

```suggestion
                  "type": "string",
                  "enum": ["processing", "failed", "succeed"]
```

The good point is that you can also remove the status you do NOT expect in the `index` action (maybe we don't want to return the keywords that are `processing`).
it's an additional way to test 👍

---

I think you made the changes in 1 out of 2 of the objects... You can see the lines referred by the 2 comments are unchanged (but it's implemented later in the file ^^)

---

Nice catch, fixed it 🙏